iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -2003,15 +2003,15 @@
 /* These translations are used for all type.highway.*.bridge. */
 "type.highway.footway.bridge" = "Puente";
 
-"type.highway.footway.demanding_alpine_hiking" = "Camino";
+"type.highway.footway.demanding_alpine_hiking" = "Path";
```

Comment:
translations are replaced again!

Reply:
aaaaargh! damn it.

Reply:
What is your system locale? Which ruby version are you using?

Reply:
locale is en-GB, Ruby version is 3.2.2

Reply:
It looks like a bug in our patched twine. Can you try to log/debug it?

Reply:
i'll try to 👍